capstone/src/main/java/com/google/univiz/CollegeId.java

```diff
@@ -0,0 +1,10 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/** CollegeId is a dummy class just to override compilation errors. */
+@AutoValue
+abstract class CollegeId {
+
+  abstract int id();
```

Comment:
will collegeId only have ids or will there also be a string attached?

Reply:
not sure. thats up to whoever is using the class, i wasnt sure what it needed.